script/Arcueid/Day 2/02_00_ARC02_1.txt

```diff
@@ -357,7 +357,7 @@ However―――
 [sha:385490faa541e70e5790e052be63927aa013436c]{
 -- Page 24, Offset 2664.
 -- 「……しまった。昨日のうちに言っておくべきだった」
-"...Oh no. I should have told you yesterday."
+"I'm sorry. I should have told you yesterday."
```

Comment:
Probably works  more like "my bad" or something.

Reply:
Yeah, that's the vibe I read too, but starting the topic with 'my bad' feels off to me. More something you'd say once it's already established what's going on. "Ah, crap" was my first thought but no swearing at the dinner table. "Ah, shoot"? Or too charlie brown?